src/components/inputs/filled-icon-input/index.jsx

```diff
@@ -0,0 +1,33 @@
+import { TextField, InputAdornment } from "@mui/material";
+
+const FilledIconInput = ({
+  icon = null,
+  placeHolder = "place holder",
```

Comment:
placeholder is one word 
change this please : "place holder"

Reply:
I will change it, thanks.